containerd/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ghcr.io/westonsteimel/containerd:1.6.1 as source
+
+FROM alpine
+COPY --from=source /usr/local/bin/ /usr/local/bin/
```

Comment:
If we copy only containerd binary, doesn't it work?

Reply:
this folder contains next files:
```sh
$ ls /usr/local/bin
containerd
containerd-shim
containerd-shim-runc-v1
containerd-shim-runc-v2
containerd-stress
```

so it seems yes, we need only `containerd`, isn't it?